Day-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+
+function ImageThumbnail(props) {
+    return (
+      <React.Fragment>
+        <img src="https://ibin.co/3whrpKSBbZ81.png" />
+      </React.Fragment>
```

Comment:
I'm fairly certain you could forego the fragment here since `img` is just one element:
```javascript
return <img src="..." />;
```